lending_pool/src/router.rs

```diff
@@ -55,15 +55,19 @@ pub trait RouterModule: proxy::ProxyModule + factory::FactoryModule {
     fn upgrade_liquidity_pool(
         &self,
         base_asset: TokenIdentifier,
-        new_bytecode: BoxedBytes,
+        new_bytecode: ManagedBuffer,
     ) -> SCResult<()> {
         require!(
             self.pools_map().contains_key(&base_asset),
             "no pool found for this asset"
         );
 
-        let pool_address = self.pools_map().get(&base_asset).unwrap();
-        let success = self.upgrade_pool(&pool_address, &new_bytecode);
+        let pool_address = self
+            .pools_map()
+            .get(&base_asset)
+            .unwrap_or_else(|| self.types().managed_address_zero());
```

Comment:
Should be able to use ManagedAddress::zero(), same as in other similar places.

Reply:
done